src/test/java/com/PlaceLab/HarisJasarevic/qamp/pages/LoginPage.java

```diff
@@ -16,13 +16,15 @@ public class LoginPage {
     private final static String EXPECTED_ERROR_MSG = "Invalid credentials!";
 
     private final WebDriver driver;
+    private BasePage basePage;
```

Comment:
base page should be extended, and must not be instantiated in the page object classes, it should be abstract

Reply:
Thanks a lot Nejla for your time, I'll make the changes.